python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -36,27 +36,37 @@ def __call__(self, catalog):
 
 
 class StarGalaxyLabeller(object):
-    labels = {"star": 0, "galaxy": 1}
-    plot = ["star", "galaxy"]
+    labels = {"star": 0, "galaxy": 1, "unknown": 9}
+    plot = ["star", "galaxy", "unknown"]
     _column = "base_ClassificationExtendedness_value"
 
     def __call__(self, catalog):
-        return np.where(catalog[self._column] < 0.5, 0, 1)
+        starGal = catalog[self._column]
+        starGal[np.isnan(starGal)] = 9
+        starGal[(starGal > 0.5) & (starGal < 1.5)] = 1
+        starGal[starGal <= 0.5] = 0
+        return starGal  # np.where(catalog[self._column] < 0.5, 0, 1)
 
 
 class OverlapsStarGalaxyLabeller(StarGalaxyLabeller):
-    labels = {"star": 0, "galaxy": 1, "split": 2}
-    plot = ["star", "galaxy", "split"]
+    labels = {"star": 0, "galaxy": 1, "split": 2, "unknown": 9}
+    plot = ["star", "galaxy", "split", "unknown"]
 
     def __init__(self, first="first_", second="second_"):
         self._first = first
         self._second = second
 
     def __call__(self, catalog1, catalog2=None):
         catalog2 = catalog2 if catalog2 else catalog1
-        first = np.where(catalog1[self._first + self._column] < 0.5, 0, 1)
-        second = np.where(catalog2[self._second + self._column] < 0.5, 0, 1)
-        return np.where(first == second, first, 2)
+        starGal1 = catalog1[self._first + self._column]
```

Comment:
Some documentation or comments would be useful to understand why this is doing this and how the deliminating values were chosen.

Reply:
I've added a comment (and hopefully anyone needing a bit more context will read the commit message).